src/main/kotlin/carracing/game/domain/data/Car.kt

```diff
@@ -0,0 +1,14 @@
+package carracing.game.domain.data
+
+private const val INITIAL_POSITION = 0
+private const val MIN_NUMBER_TO_MOVE = 4
+
+data class Car(
+    var position: Int = INITIAL_POSITION,
```

Comment:
How about making it mutable only within the `Car` class, and read-only from the outside?